libjpeg-turbo.yaml

```diff
@@ -64,5 +64,3 @@ update:
   enabled: true
   github:
     identifier: libjpeg-turbo/libjpeg-turbo
```

Comment:
Was pulling beta tags. Didn't spot any reason why this was using tags and limiting to 3.x release when this is not version streamed.

By using releases, it'll filter out the beta releases

Reply:
agree 3.0.90 cut is pre-release and using github releases would exclude it. Thanks for the fix Mark.